osm2pgsql.lua

```diff
@@ -499,128 +499,116 @@ function osm2pgsql.process_node(object)
         object.tags.wikidata == 'Q994730') then
         output_hstore['disputed_by'] = 'CN;RU;IN;GR'
     end
--- Recast Northern Cyprus as country label and turn off for several POVs including China and Russia
+
+-- Turn off Northern Cyprus label
     if object.tags.place and object.tags.wikidata == 'Q23681' then
-        output_hstore['place'] = 'country'
-        output_hstore['disputed_by'] = 'CN;RU;IN;GR;CY'
+        output_hstore['place'] = nil
     end
--- Turn off Abkhazia label for most countries
+-- Turn off Abkhazia label
     if object.tags.place and object.tags.wikidata == 'Q23334' then
-        output_hstore['place:AR'] = 'region'
-        output_hstore['place:BD'] = 'region'
-        output_hstore['place:BR'] = 'region'
-        output_hstore['place:CN'] = 'region'
-        output_hstore['place:DE'] = 'region'
-        output_hstore['place:EG'] = 'region'
-        output_hstore['place:GB'] = 'region'
-        output_hstore['place:GR'] = 'region'
-        output_hstore['place:ID'] = 'region'
-        output_hstore['place:IL'] = 'region'
-        output_hstore['place:IN'] = 'region'
-        output_hstore['place:IT'] = 'region'
-        output_hstore['place:JP'] = 'region'
-        output_hstore['place:KO'] = 'region'
-        output_hstore['place:MA'] = 'region'
-        output_hstore['place:NL'] = 'region'
-        output_hstore['place:NP'] = 'region'
-        output_hstore['place:PK'] = 'region'
-        output_hstore['place:PL'] = 'region'
-        output_hstore['place:PS'] = 'region'
-        output_hstore['place:PT'] = 'region'
-        output_hstore['place:SA'] = 'region'
-        output_hstore['place:SE'] = 'region'
-        output_hstore['place:TR'] = 'region'
-        output_hstore['place:TW'] = 'region'
-        output_hstore['place:UA'] = 'region'
-        output_hstore['place:VN'] = 'region'
+        output_hstore['place'] = nil
     end
--- Turn off South Ossetia label for most countries
+-- Turn off South Ossetia label
     if object.tags.place and object.tags.wikidata == 'Q23427' then
-        output_hstore['place:AR'] = 'region'
```

Comment:
Here South Ossetia isn't a region label for Georgia... so I think you want to set the default place tag to country so it'll get exported as `unrecognized`...

But you'll want to add **place:RU** = `country` since Russia recognizes it as a country.

Remove the others.